nifi-nar-bundles/nifi-aws-bundle/nifi-aws-nar/pom.xml

```diff
@@ -32,13 +32,14 @@
     <dependencies>
         <dependency>
             <groupId>org.apache.nifi</groupId>
-            <artifactId>nifi-standard-services-api-nar</artifactId>
+            <artifactId>nifi-aws-service-api-nar</artifactId>
+            <version>${project.version}</version>
             <type>nar</type>
         </dependency>
         <dependency>
             <groupId>org.apache.nifi</groupId>
             <artifactId>nifi-aws-processors</artifactId>
-            <version>1.4.0-SNAPSHOT</version>
```

Comment:
I'm not well informed on the pros and cons of using the project.version variable as opposed to the literal version number.  The NiFi project as a whole manages literal version numbers with the Maven Release Plugin , so the variables are not necessary to keep references consistent.  What has been your experience?

Reply:
It's a style choice, one I follow out of habit. The release plugin is perfectly happy with such references, and I find it clearer to declare the version once at the the parent pom reference and stick with it, unless the version is deliberately varying. But that's for your project to decide, so happy to revert my habit here if wanted.

Reply:
We actually had a problem in the past with the release plugin and project.version, I can't remember the specifics and it may very well not be an issue anymore, but we did specifically make the choice to use the literal version everywhere so I'd prefer to stick with that